bootstrap/kubeadm/controllers/kubeadmconfig_controller_test.go

```diff
@@ -229,43 +222,26 @@ func TestKubeadmConfigReconciler_Reconcile_MigrateToSecret(t *testing.T) {
 	}
 
 	result, err := k.Reconcile(request)
-	if err != nil {
-		t.Fatalf("failed to reconcile: %v", err)
-	}
-	if result.Requeue == true {
-		t.Fatal("did not expect to requeue")
-	}
-	if result.RequeueAfter != time.Duration(0) {
-		t.Fatal("did not expect to requeue after")
-	}
-
-	if err := k.Client.Get(context.Background(), client.ObjectKey{Name: config.Name, Namespace: config.Namespace}, config); err != nil {
-		t.Fatalf("failed to get KubeadmConfig: %v", err)
-	}
+	g.Expect(err).NotTo(HaveOccurred())
+	g.Expect(result.Requeue).To(BeFalse())
+	g.Expect(result.RequeueAfter).To(Equal(time.Duration(0)))
 
-	if config.Status.DataSecretName == nil {
-		t.Fatal("expected bootstrap data secret to be populated")
-	}
+	err = k.Client.Get(context.Background(), client.ObjectKey{Name: config.Name, Namespace: config.Namespace}, config)
```

Comment:
this (and all similar functions that return only an error) can get condensed into

`g.Expect(k.Client.Get(context.Background(), client.ObjectKey{Name: config.Name, Namespace: config.Namespace}, config)).To(Succeed())

Reply:
made the change